infra/Pos/Network/Types.hs

```diff
@@ -1,5 +1,6 @@
+{-# LANGUAGE CPP                       #-}
 {-# LANGUAGE ExistentialQuantification #-}
-{-# LANGUAGE CPP #-}
+{-# LANGUAGE OverloadedStrings         #-}
```

Comment:
In `infra` (and I guess in all other packages) this extension is enabled by default too.